scripts/partition_manager_output.py

```diff
@@ -52,6 +52,7 @@ def add_line(text_before_split, text_after_split):
                                       key=lambda key_value_tuple: key_value_tuple[1]['address']):
 
             add_line("%s_ADDRESS" % name.upper(), "0x%x" % partition['address'])
+            add_line("%s_END_ADDRESS" % name.upper(), "0x%x" % partition['end_address'])
```

Comment:
Could you add a comment here stating that this is here to facilitate generator expressions? Otherwise people (me in the future) might start complaining.

Reply:
Done